draft-ietf-tls-external-psk-importer.md

```diff
@@ -91,15 +90,14 @@ when, and only when, they appear in all capitals, as shown here.
 Key importers mirror the concept of key exporters in TLS in that they diversify a key
 based on some contextual information before use in a connection. In contrast to key exporters,
 wherein differentiation is done via an explicit label and context string, the key importer
-defined herein uses an optional context string along with a target protocol and KDF
-identifier to differentiate an external PSK into one or more PSKs for use.
+defined herein diversifies external one PSK into one or more PSKs for use via a target protocol,
+KDF identifier, and optional context string. Additionally, the resulting PSK binder key is modified
+with a new derivation label to prevent confusion with non-imported PSKs.
 
 Imported keys do not require negotiation for use, as a client and server will not agree upon
-identities if not imported correctly. Thus, importers induce no protocol changes with
-the exception of expanding the set of PSK identities sent on the wire. Endpoints may
-incrementally deploy PSK importer support by offering non-imported keys for TLS versions
-prior to TLS 1.3. Non-imported and imported PSKs are distinct since their identities are
-different on the wire. See {{rollout}} for more details.
+identities if not imported correctly. Endpoints may incrementally deploy PSK importer support
+by offering non-imported keys for TLS versions prior to TLS 1.3. Non-imported and imported PSKs
+are distinct since their identities are different on the wire. See {{rollout}} for more details.
```

Comment:
I don't think that there is anything forcing identities to be distinct, other than perhaps the normative force of a statement in this document.  A non-imported PSK could use an identity that collides with an imported one.

Reply:
True, though this sentence is primarily about collisions across identities given the same input PSK.